src/css/HomePage.css

```diff
@@ -0,0 +1,97 @@
+/* All h1 text elements. In this case, it's just the "The Jumpstart Blog" text. */
+h1 {
+  margin-left: 30px;
```

Comment:
Seems weird to put set margins on element selectors. Can we just us a title class?